css/src/atomics/ratio.scss

```diff
@@ -0,0 +1,8 @@
+@each $name, $ratio in $ratios {
+	.ratio-#{$name} {
+		aspect-ratio: #{$ratio};
+		width: 100% !important; // must override width to fill ratio
+		height: 100% !important; // must override height to fill ratio
+		object-fit: cover;
```

Comment:
As a thought, would it be worth introducing an `object-fit-cover` atomic, so folks can opt into using that? That way, these ratio atomics can do somewhat fewer things at once.

Reply:
We can do this but it will stretch the image (same behavior as the aspect ratio in mdn). Maybe we can just use video examples?

Reply:
Makes sense for atomics, I'll do this in a separate PR.